### DISCUSSION TITLE
easymotion like integration

### DISCUSSION THREAD

**Discussion by spbeach46** [gdh1995/vimium-c#794] [Ideas]
I do like the much improved 'yv' for locating lines that you're most
likely to highlight and and then jumping directly into visual mode for
yanking, etc. It's very intuitive. But something like vim-easymotion (via
https://github.com/easymotion/vim-easymotion) would be amazing for
consistent accuracy doing so.

**Comment by gdh1995** [comment]
Um what detailed actions do you want Vimium C to do? EasyMotion is too big to simulate it in a browser.

**Comment by spbeach46** [comment]
Just simulating the most basic functionality of easymotion in a vimium way. I didn't imagine a full integration. But say 'ye' opens up a prompt for single character search much like '/'. With matches it would function just like 'yv'; displaying 2-letter codes, jumping to, and going into to visual mode. It would be a nice middle ground between the 'yv' and '/' --> search --> '<enter>' --> 'v' sequence. Apologies for any assumptions I'm making. This is my first discussion post ever. I imagine a ton of work goes into this project.Easily my favorite firefox/chrome extension.

**Comment by gdh1995** [comment]
Sounds useful but indeed a heavy work.

Because Vimium C is being upgrading into a format of Manifest V3 (currently it's in M2), I won't implement these features in a recent year. Sorry but my work in the furture months will focus on `bug fixes` in the new extension platform, supports for `tab groups` of Chrome and then improvement on `Vomnibar usages`.

Currently there's a short path about `/`: when you're in Visual mode and press `/` to show the Find HUD, then a `<enter>` key will make Find HUD hide and then Vimium C will auto re-enter Visual mode . Of course your idea like `ye` + `yv` and `/` + search + `yv` will help to make a work flow shorter.
